package.json

```diff
@@ -164,7 +164,20 @@
           "when": "never"
         }
       ]
-    }
+    },
+    "configuration": [
+      {
+          "title": "Azure Resource Explorer",
+          "properties": {
+              "azure-resource-explorer-for-vscode.customApiVersions": {
+                  "scope": "resource",
+                  "type": "array",
+                  "default": [],
+                  "description": "Allows to override 'api-version' values to be used for specific paths. An array of {'path': '/path/to/resource', 'version': 'api-version-to-use'} objects."
```

Comment:
i thought it might be more user- and typescript-friendly to have an array of objects with two properties.